scripts/verify.ps1

```
@@ -0,0 +1,12 @@
+$ErrorActionPreference = 'Stop'
+
+$repoRoot = Split-Path -Parent $PSScriptRoot
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'repoRoot' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)